src/main/java/com/palyrobotics/frc2020/robot/Robot.java

```diff
@@ -67,6 +69,11 @@
 			new TelemetryService()),
 			mEnabledServices;
 
+	private NetworkLoggerService mNetworkLogger = new NetworkLoggerService();
```

Comment:
As the amount of services are getting larger it might be a good idea to put them all into a list to make it cleaner